.travis.yml

```diff
@@ -16,12 +15,5 @@ install:
 
 script:
    - "export EXCEPT_DEPLOYED=true"
-   - "if [[ $TRAVIS_RUBY_VERSION == '2.4.2' ]]; then export EXCEPT_DEPLOYED=false; fi"
+   - "if [[ $TRAVIS_RUBY_VERSION == '2.5.8' ]]; then export EXCEPT_DEPLOYED=false; fi"
```

Comment:
Why do we need to test against specific minor Ruby version? Does this condition apply to all 2.5.x rubies?

Reply:
Gotcha, with Github actions we can specify the matrix with just:
```
matrix:
        ruby-version: ['2.5', '2.6', '2.7']
```

Reply:
Makes sense! I actually made all the tests have `except_deployed` be false unless it's a push directly to master.

The current GitHub actions should all be the looser requirements now. I think we just need to specify more specific gemspec dependency versions.

I have a few AWS permission bugs to workout but were very close to pushing an update!